src/test/config/bitbucket/values-KITT.yaml

```diff
@@ -30,10 +30,3 @@ bitbucket:
   mesh:
```

Comment:
I think we could remove the whole mesh section, but this is good as well.

Reply:
We need pod annotations since the plan that tests mesh needs it when Bitbucket is deployed to KITT.